.github/workflows/release.yml

```
@@ -32,3 +32,17 @@ jobs:
           workflow: publish.yml
           ref: master
           inputs: '{ "acvm-ref": "${{ needs.release-please.outputs.tag-name }}" }'
+
+  dispatch-publish-acvm-js:
+    name: Dispatch to publish-acvm-js workflow
+    needs: [release-please]
+    if: ${{ needs.release-please.outputs.tag-name }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger publish-acvm-js.yml workflow
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: publish-acvm-js.yml
+          repo: ${{ github.repository }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

```suggestion
          ref: master
```

Nit, but for consistency with the job above.